arborist/user.go

```diff
@@ -15,6 +15,7 @@ type User struct {
 }
 
 type UserFromQuery struct {
+	ID       int64          `db:"id"`
```

Comment:
do the queries below (like in `userWithName`) need to be updated to get `SELECT usr.id, usr.name ...` ?

Reply:
this might be able to be uint32, seems like `serial` type in postgres is 32 bits https://www.postgresql.org/docs/current/datatype-numeric.html

Reply:
there are some uses of `UserFromQuery` in the DEVELOP.md as well, may want to update those too

Reply:
sure, will update those. weird, I've seen different datatypes used for this. I'll check what it should be. there would be a lot of structs to change so might want to have that be a separate thing regardless.